src/Orckestra.Composer.CustomerConditionProvider/Providers/CustomerConditionProvider.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Orckestra.Composer.CustomerConditionProvider.Repositories;
+using Orckestra.Composer.Parameters;
+using Orckestra.Composer.Repositories;
+using Orckestra.Composer.Services;
+using Orckestra.Overture.ServiceModel.Customers;
+using Orckestra.Overture.ServiceModel.Metadata;
+using Orckestra.Tools.ConditionalContent.Providers;
+using Orckestra.Tools.ConditionalContent.Types.Config;
+
+namespace Orckestra.Composer.CustomerConditionProvider.Providers
+{
+    public class CustomerConditionProvider : IConditionProvider
+    {
+        protected IComposerContext ComposerContext { get; }
+        protected ICustomerRepository CustomerRepository { get; }
+        protected ICustomerDefinitionsRepository CustomerDefinitionsRepository { get; }
+
+        public CustomerConditionProvider(IComposerContext composerContext,ICustomerRepository customerRepository, ICustomerDefinitionsRepository customerDefinitionsRepository)
+        {
+            ComposerContext = composerContext;
+            CustomerRepository = customerRepository ?? throw new ArgumentNullException(nameof(customerRepository));
+            CustomerDefinitionsRepository = customerDefinitionsRepository ?? throw new ArgumentNullException(nameof(customerDefinitionsRepository));
+        }
+
+        private Dictionary<string, List<string>> Fields => new Dictionary<string, List<string>>()
```

Comment:
if this collection never changes it should be "private static readonly"

Reply:
require C# 8.0